Framework/DataHandling/src/LoadILLTOF2.cpp

```diff
@@ -72,6 +72,7 @@ void LoadILLTOF2::init() {
   declareProperty(std::make_unique<WorkspaceProperty<>>("OutputWorkspace", "",
                                                         Direction::Output),
                   "The name to use for the output workspace");
+  declareProperty("ConvertToTOF", false, Direction::Input);
```

Comment:
We need a description here of the ConvertToTof property, this is used as the tooltip, and also in the documentation.